README.md

```
@@ -15,52 +15,69 @@ GitLab のロゴ、タヌキマスコット、ブランドカラーは GitLab In
 
-- [Astro](https://astro.build/) — 静的サイト生成
-- Cloudflare Pages — ホスティング
+- [Hugo](https://gohugo.io/) (extended 0.151.0) — 静的サイト生成。upstream と同じテーマ ([docsy](https://www.docsy.dev/) v0.12.0 + [docsy-gitlab](https://gitlab.com/gitlab-com/content-sites/docsy-gitlab) v0.3.67) を Hugo Modules で参照
+- Cloudflare Pages — ホスティング (direct-upload)
 - Cloudflare R2 — 画像配信
-- [Anthropic Claude API](https://docs.claude.com/) — 翻訳エンジン
+- [Anthropic Claude](https://docs.claude.com/) — 翻訳エンジン (Claude Code の `translator` サブエージェント経由)
 
 ## ディレクトリ構成
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)